pkg/kubectl/cmd/create/create_tenant.go

```diff
@@ -0,0 +1,93 @@
+/*
+Copyright 2015 The Kubernetes Authors.
```

Comment:
Please use "Arktos Authors" in the copyright header for new files.

Reply:
Fixed.